helpers/config.py

```diff
@@ -780,6 +793,50 @@ def __questions_aws(self):
             self.__dict['aws_secret_key'] = ''
             self.__dict['aws_bucket_name'] = ''
 
+    def __questions_aws_validate_credentials(self):
+        """
+        Prompting user whether they would like to validate their entered AWS
+        credentials or continue without validation.
+        """
+        aws_credential_attempts = 0
+
+        if self.__dict['use_aws']:
+            self.__dict['aws_validate_credentials'] = CLI.yes_no_question(
+                'Would you like to validate your AWS credentials?',
+                default=self.__dict['aws_validate_credentials'],
+            )
+
+        if self.__dict['use_aws'] and self.__dict['aws_validate_credentials']:
+            while not self.__dict['aws_credentials_valid'] and (
+                aws_credential_attempts <= self.MAXIMUM_AWS_CREDENTIAL_ATTEMPTS
+            ):
+                aws_credential_attempts += 1
+                self.validate_aws_credentials()
+                attempts_remaining = self.MAXIMUM_AWS_CREDENTIAL_ATTEMPTS - \
+                        aws_credential_attempts
+                if (
+                    not self.__dict['aws_credentials_valid']
+                    and attempts_remaining >= 0
+                ):
+                    CLI.colored_print(
+                        'Invalid credentials, please try again.\n'
+                        'Attempts remaining for AWS validation: {}'.format(
+                            attempts_remaining
```

Comment:
I think we should use `attempts_remaining + 1` 

```shell 
AWS Access Key []:
AWS Secret Key []:
AWS Bucket name []:
Would you like to validate your AWS credentials?
	1) Yes
	2) No
[2]: 1
Invalid credentials, please try again.
Attempts remaining for AWS validation: 1  <-- Actually, 2 are remaining at this moment.
AWS Access Key []:
AWS Secret Key []:
AWS Bucket name []:
Invalid credentials, please try again. 
Attempts remaining for AWS validation: 0  <-- Only one remains
AWS Access Key []:
AWS Secret Key []:
AWS Bucket name []:
```

Another though, do you think it could be a good idea to use WARNING or ERROR for `Invalid credentials, please try again.` and INFO for `Attempts remaining for AWS validation:`?